### ISSUE TITLE
Opening Desktop Client on MacOS always locks Safari Browser Extension

### ISSUE THREAD

**Issue by zexpe** [bitwarden/clients#6763]
### Steps To Reproduce

1. Open Safari
2. Unlock Bitwarden Safari Browser Extension
3. Open Bitwarden Desktop Client App
4. This opens a new tab in Safari: https://bitwarden.com/browser-start/
5. This also locks the Bitwarden Safari Browser Extension
6. Steps 2-5 can then be repeated several times over

### Expected Result

Point 6 certainly shouldn't happen, but I'm also confused why points 4 & 5 happen at all, even once.

### Actual Result

Every time the Desktop Client is opened it locks the browser extension and opens the https://bitwarden.com/browser-start/ URL.

### Additional Context

This problem has persisted for many releases.

### Operating System

macOS

### Operating System Version

14.1

### Installation method

Mac App Store

### Build Version

2023.10.0

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by NovaSilentium** [comment]
Hi there,

I am unable to reproduce this issue, it has been escalated for further investigation. If you have more information that can help us, please add it below. 

Thanks!

**Comment by zexpe** [comment]
It happens for a while after you have first updated Bitwarden Desktop Client to the latest version, eventually after enough times opening the app, or enough time has passed, it disappears.

**Comment by ywwry66** [comment]
This has been happening since a few versions ago. See #5094

**Comment by zexpe** [comment]
Yes, it's been happening for at least a year in fact. After every update. I suspect the App Store update is updating the Safari Extension and Desktop Client, but upon opening the Desktop Client it is then reinstalling the Safari Extension. If the extension is already installed it shouldn't be doing this at all. Why it does it on every launch for sometime before the behaviour eventually ends is also odd.

**Comment by saschafoerster** [comment]
We are a team of 5 people experiencing the same. If I can help debug, I am ready. For it seems like after some the extension closes itself, then the browser opens this URL: https://bitwarden.com/browser-start/

**Comment by zexpe** [comment]
Curiously this bug didn't happen for me today with the 2023.10.1 update - however, unusually I also rebooted my computer between updating Bitwarden and opening it, to apply the macOS 14.1.1 update. I don't know if that makes the difference?

**Comment by ywwry66** [comment]
Unfortunately, I am still having the issue. Usually a reboot does not resolve the problem for me, which I have tried several times before. But maybe the reboot after a macOS update is somewhat different.

**Comment by closebot-bw** [comment]
⚠️ **Stale Issue Notice**

This issue has been automatically marked as stale due to inactivity. It will be closed in **2 weeks** (October 15, 2025) if no further activity occurs.

If this issue is still relevant and you would like to keep it open, please:
- Comment on this issue to show continued interest
- Provide any additional information or updates
- Confirm that the issue still exists in the latest version

Thank you for your contribution to this project! 🙏

**Comment by closebot-bw** [comment]
🔒 **Issue Closed Due to Inactivity**

This issue has been automatically closed due to lack of activity for an extended period. We periodically review and close inactive issues to help maintain our issue tracker and focus on current priorities.

**If this issue is still relevant:**
- Please create a new issue with updated information
- Include steps to reproduce the problem if it's a bug report
- Mention if this issue still occurs in the latest version

Thank you for your contribution to this project. Your feedback helps us improve! 🙏
